packages/react-ui-components/src/SelectBox/selectBox.js

```diff
@@ -185,6 +185,7 @@ export default class SelectBox extends PureComponent {
                         <SelectBox_ListPreview
                             {...this.props}
 
+                            theme={theme}
```

Comment:
Is this actually needed / correct? I think we never should need this; but the corresponding styles should be in SelectBox_ListPreview/style.css then